msktutil.cpp

```diff
@@ -835,6 +843,7 @@ int main(int argc, char *argv [])
         if (!strcmp(argv[i], "--service") || !strcmp(argv[i], "-s")) {
             if (++i < argc) {
                 exec->add_principals.push_back(argv[i]);
+               flags->add_only_default_spns = false;
```

Comment:
Unless the web interface fools me, this looks like a mismatched indent.

Reply:
fixed. thanks